docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,7 @@ develop
     *    - Type
          - Change
 
+=======
```

Comment:
What's this separator doing?

Reply:
The release notes got messed up, so just reset it to the develop version